app/controllers/profiles_controller.rb

```diff
@@ -63,11 +57,23 @@ def profile_params
   end
 
   def user_params
-    params.require(:user).permit(:display_name)
+    params.require(:user).permit(:display_name, :unique_name)
+  end
```

Comment:
Don't really think these are being used? Maybe tho

Reply:
In the profile edit action you can update your username. My thinking is that the profile represents the user's public face, so editing the display name through the same form makes sense. As opposed to an email address.